### PR TITLE
🐛PRTL-690 filter most frequent mutations by project

### PR DESCRIPTION
**Pull request by cheapsteak**
Project Entity page
Most Frequent Mutations and Most Frequently Mutated Genes now accounting for project id when making request

Case Entity Page
Most Frequent Mutations accounting for case id when making request

### CODE REVIEW

**Comment by cheapsteak** [comment]
@Jephuff care to review this? touches mostly your code (but very minimal amount of code touch)

File: modules/node_modules/@ncigdc/components/Case.js [hunk 1]
```diff
@@ -281,13 +282,14 @@ const Case = ({
         <LocationSubscriber>{(ctx: {| pathname: string, query: TRawQuery |}) => {
           const { filters } = ctx.query || {};
           const currentFilters = parseFilterParam(filters, { op: 'and', content: [] });
+          const componentFilters = _.mergeWith(currentFilters, setFilter({ field: 'cases.case_id', value: p.case_id }), (a, b) => (Array.isArray(a) ? a.concat(b) : undefined));
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/components/Case.js]: Might be good to using the methods in the filters utils. something like this https://github.com/NCI-GDC/portal-ui/blob/relay/modules/node_modules/%40ncigdc/components/Oncogrid/getQueries.js#L19

**Comment by cheapsteak** [comment]
Thanks for the usage example! That looks so much better @Jephuff ready for re-review?

**Comment by cheapsteak** [comment]
@Jephuff Thank you!

**Pull request merged**